pc/request.cpp

```diff
@@ -295,6 +295,20 @@ str product::get_symbol()
   return sym;
 }
 
+str product::get_base()
+{
+  str base;
+  get_attr( attr_id( "base" ), base);
+  return base;
+}
+
+str product::get_quote()
```

Comment:
Should really be named `get_quote_currency` since `get_quote` implies something entirely different